ui_testing/testcases/basic_tests/test04_testunits.py

```diff
@@ -933,111 +933,75 @@ def test026_edit_approach_overview_button(self):
         self.assertEqual(self.base_selenium.get_url(), '{}testUnits'.format(self.base_selenium.url))
         self.base_selenium.LOGGER.info('clicking on Overview confirmed')
 
-    @parameterized.expand(['Quantitative', 'Qualitative', 'MiBi'])
-    def test_027_changing_testunit_type_update_fields_accordingly(self, testunit_type):
+    def test_027_changing_testunit_type_update_fields_accordingly(self):
         """
-        New: Test unit: Type Approach: When I change type from edit mode, the values should changed according to this type that selected 
-        When I change type from edit mode, the values should changed according to this type that selected 
-        LIMS-3680
-
-        comment: this case will be handled in create
-        """
-
-        if testunit_type == 'MiBi':
-            testunit_type = 'Quantitative MiBi'
-
-        self.base_selenium.LOGGER.info('open testunits in create')
-        self.test_unit_page.click_create_new_testunit()
+        New: Test unit: Type Approach: When I change type from edit mode, the values
+        should changed according to this type that selected
 
-        self.base_selenium.LOGGER.info('set the type to {}'.format(testunit_type))
-        self.test_unit_page.set_testunit_type(testunit_type=testunit_type)
-        self.test_unit_page.sleep_tiny()
-        self.base_selenium.LOGGER.info(
-            'set testunit type to {}, fields should be displayed as the following'.format(testunit_type))
-
-        if testunit_type == 'Quantitative':
-            self.assertTrue(self.test_unit_page.check_for_quantitative_fields())
-        elif testunit_type == 'Qualitative':
-            self.assertTrue(self.test_unit_page.check_for_qualitative_fields())
-        elif testunit_type == 'Quantitative MiBi':
-            self.assertTrue(self.test_unit_page.check_for_quantitative_mibi_fields())
-
-    def test_028_allow_user_to_change_between_specification_and_quantification(self):
+        LIMS-3680
         """
-        New: Test unit: Edit mode:  Limit of quantification Approach: Allow user to change between the two options specification and limit of quantification from edit mode.
-        Allow user to change between the two options specification and limit of quantification from edit mode.
-        specification to quantifications
+        self.info('edit random testunit')
+        self.test_unit_page.get_random_test_units()
+        self.base_selenium.get_url()
+        testunit_types = ['Quantitative', 'Qualitative', 'Quantitative MiBi']
```

Comment:
Revirt those changes, for loop means all of them are one test case >> One fail, all fail

However, paramterized means they r n  test  cases

Reply:
ok

Reply:
@ghada-37 
- both links are not working
- Could u please parameterized them and run them in travis?

Reply:
ok